docs/reference/formulas/smartwatts.md

```diff
@@ -1,17 +1,13 @@
 # SmartWatts Formula
 
-SmartWatts is a software-defined power meter based on the PowerAPI toolkit.
-SmartWatts is a configurable software that can estimate the power consumption of
-software in real-time.
-SmartWatts needs to receive several metrics provided by
-[HWPC Sensor](../sensors/hwpc-sensor.md#events) :
+SmartWatts is a Formula, a configurable software that can estimate the power consumption of software in real-time.
+SmartWatts needs to receive several metrics available in [Reports](../report/report.md), [HWPC Sensor](../sensors/hwpc-sensor.md#events) is a Sensor compatible (i.e making the necessary metrics available in HWPC Reports):
 
 - The Running Average Power Limit (`RAPL`)
 - `msr` events (`TSC`, `APERF`, `MPERF`)
 - `core` events which depend on the Processor Architucture
 
-These metrics are then used as inputs for a power model that estimates the power
-consumption of each software.
+These metrics are then used as inputs for a power model that estimates the power consumption of each software, those estimations are recorded in Power Reports.
```

Comment:
I would say : These raw metrics are then used as inputs for a power model that estimates the power consumption of each software/process(?). These estimations are recorded in `Power Reports` (reference to PowerReports..).